src/language/tekster_en.ts

```diff
@@ -410,6 +410,9 @@ export default {
   'arbeidsforhold.knapp.slettArbeidsgiver': 'Remove employer',
   'arbeidsforhold.label.navn': 'Your place of work',
   'arbeidsforhold.label.arbeidsmengde': 'How much do you work?',
+  'arbeidsforhold.label.arbeidsmengde.beskrivelse':
+    'If your work contract differs from what you actually work, you state how much you actually work. <br /><br />' +
+    'If it varies from month to month how much you work, estimate approximately how much you expect to work in the future',
```

Comment:
Forslag: 

- If your actual work(evt. workload/ work hours) is different from the information in the contract, state how much you actually work.
- If the workload /hours varies on a montly basis, estimate approximately how much you expect to work in the future.

Reply:
Kan evt bruke stillingsprosent (vacancy rate) "If your actual vacancy rate differs from the vacancy rate stated in the contract" ellerno